resources/tests/analysisTools/qcPipeline/chrom-sizes-file.tsv

```diff
@@ -0,0 +1,5 @@
+1	1
```

Comment:
5 Lines? That right?

Reply:
This file was for the grouped qc summary that is from 1.3.0. I forgot to delete the file after backporting the whole test code from there.